Libraries/LibWeb/HTML/Dates.cpp

```diff
@@ -414,13 +414,16 @@ static Optional<HourMinuteSecond> parse_a_time_component(GenericLexer& input)
     if (!maybe_hour.has_value())
         return {};
     auto hour = maybe_hour.value();
+
     // 2. If hour is not a number in the range 0 ≤ hour ≤ 23, then fail.
     if (hour < 0 || hour > 23)
         return {};
+
```

Comment:
my old man eyes thank you for this :pray:

Reply:
Yeah I struggle a little without the newlines 😅